indico/modules/events/editing/client/js/components/FileDisplay/FileDisplay.module.scss

```diff
@@ -0,0 +1,20 @@
+@import 'base/palette';
+
+.file-display {
+    display: flex;
+
+    .file-type-display {
+        padding: 1em;
+        margin: 1em;
+        width: 33%;
```

Comment:
I think you can achieve the same with `flex: 1` which will be probably better in case there are more than 3 columns

Reply:
I think the initial idea behind this (we use the same in FileManager) is to move to another row whenever we have more than 3 items and not try to squeeze in everything in the same row.

Reply:
that sounds a perfect use case for flexbox features

Reply:
Yep, that is why I suggested `flex` attribute ;)

Reply:
But you would still need to use a percentage to specify how many items you want per line even with flexbox (ex: `flex: 0 0 33%;`), right?

Reply:
Yeah, and probably `flex-wrap` as well